lib/shared/units.js

```diff
@@ -50,33 +43,38 @@ const MEGABYTE_TO_BYTE_RATIO = 1e+6;
 const MILLISECONDS_IN_A_DAY = 86400000;
 
 /**
- * @summary Convert bytes to gigabytes
+ * @summary Convert bytes to megabytes
  * @function
  * @public
  *
  * @param {Number} bytes - bytes
- * @returns {Number} gigabytes
+ * @returns {Number} megabytes
  *
  * @example
- * const result = units.bytesToGigabytes(7801405440);
+ * const result = units.bytesToMegabytes(7801405440);
  */
-exports.bytesToGigabytes = (bytes) => {
-  return bytes / GIGABYTE_TO_BYTE_RATIO;
+exports.bytesToMegabytes = (bytes) => {
```

Comment:
Yup, it's used in `lib/gui/models/flash-state.js`, and since it deals directly with numbers to numbers it can't be replaced by the other, as it formats to a string.